dle/cfn/deploy_ec2.yaml

```diff
@@ -0,0 +1,151 @@
+Parameters:
+  InstanceTypeParameter:
+    Type: String
+    Default: "m6i.large"
+    AllowedValues:
+      - t2.micro
+      - m5.small
+      - m6i.large
+    Description: Default is m6i.large.
+  ImageIDParameter:
+    Type: String
+    Default: "ami-005f9685cb30f234b"
+    AllowedValues:
+      - ami-005f9685cb30f234b
+    Description: Amazon Linux AMI.
+  KeyIDParameter:
+    Type: String
+    Default: "globalrx_sshkey"
+  NameParameter:
+    Type: String
+    Default: "CFN_TEST"
+    Description: Value for Name Tag
+  EmailParameter:
+    Type: String
+    Default: "aeg224@g.harvard.edu"
+    Description: email for django admin user
+  VPCParameter:
+    Type: String
+    Default: "vpc-23074059"
+    Description: VPC ID
+  DBEndpointParameter:
+    Type: String
+    Default: "asdf-globalrxdb-axodhrci4xgh.cj9gkkj3it4b.us-east-1.rds.amazonaws.com"
```

Comment:
Why does the EC2 need to connect to RDS? That should probably be denied by the RDS security group upfront